src/lib/ts/conf/tsconfig.ngc.json

```diff
@@ -10,7 +10,7 @@
   "buildOnSave": false,
   "compileOnSave": false,
   "compilerOptions": {
-    "baseUrl": "AUTOGENERATED",
+    "baseUrl": ".",
```

Comment:
This is still required for secondary entry points and for people not using a custom tsconfig
https://github.com/dherges/ng-packagr/blob/master/src/lib/ng-v5/entry-point/ts/compile-ngc.transform.ts#L23